jdaviz/configs/default/plugins/data_quality/data_quality.py

```diff
@@ -0,0 +1,285 @@
+import os
+from traitlets import Any, Dict, Bool, List, Unicode, Float, observe
+
+import numpy as np
+from glue_jupyter.common.toolbar_vuetify import read_icon
+from echo import delay_callback
+from matplotlib.colors import hex2color
+
+from jdaviz.core.registries import tray_registry
+from jdaviz.core.template_mixin import (
+    PluginTemplateMixin, ViewerSelect, LayerSelect
+)
+from jdaviz.core.tools import ICON_DIR
+from jdaviz.configs.default.plugins.data_quality.dq_utils import (
+    decode_flags, generate_listed_colormap, dq_flag_map_paths, load_flag_map
+)
+
+__all__ = ['DataQuality']
+
+telescope_names = {
+    "jwst": "JWST",
+    "roman": "Roman"
+}
+
+
+@tray_registry('g-data-quality', label="Data Quality", viewer_requirements="image")
+class DataQuality(PluginTemplateMixin):
+    template_file = __file__, "data_quality.vue"
+
+    # TODO: uncomment this line before merging into main:
+    # irrelevant_msg = Unicode("Data Quality plugin is in development.").tag(sync=True)
+
+    viewer_multiselect = Bool(False).tag(sync=True)
+    viewer_items = List().tag(sync=True)
+    viewer_selected = Any().tag(sync=True)  # Any needed for multiselect
```

Comment:
could use the `ViewerSelectMixin` to remove some boilerplate code (unless there is some custom overrides I didn't notice?)

Reply:
Correct, removing. 💯